lib/screens/home/global_settings/global_settings_screen.dart

```diff
@@ -0,0 +1,269 @@
+import 'package:flutter/foundation.dart';
+import 'package:prov_wallet_flutter/prov_wallet_flutter.dart';
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/common/widgets/pw_app_bar.dart';
+import 'package:provenance_wallet/common/widgets/pw_app_bar_gesture_detector.dart';
+import 'package:provenance_wallet/common/widgets/pw_dialog.dart';
+import 'package:provenance_wallet/common/widgets/pw_list_divider.dart';
+import 'package:provenance_wallet/screens/change_pin_flow/change_pin_flow.dart';
+import 'package:provenance_wallet/screens/home/global_settings/category_label.dart';
+import 'package:provenance_wallet/screens/home/global_settings/developer_menu.dart';
+import 'package:provenance_wallet/screens/home/global_settings/future_toggle_item.dart';
+import 'package:provenance_wallet/screens/home/global_settings/link_item.dart';
+import 'package:provenance_wallet/screens/home/global_settings/toggle_item.dart';
+import 'package:provenance_wallet/screens/home/home_bloc.dart';
+import 'package:provenance_wallet/services/account_service/account_service.dart';
+import 'package:provenance_wallet/services/crash_reporting/crash_reporting_service.dart';
+import 'package:provenance_wallet/services/key_value_service/key_value_service.dart';
+import 'package:provenance_wallet/services/models/account.dart';
+import 'package:provenance_wallet/util/get.dart';
+import 'package:provenance_wallet/util/strings.dart';
+import 'package:provenance_wallet/util/timed_counter.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class GlobalSettingsScreen extends StatefulWidget {
```

Comment:
I might just go with `SettingsScreen` and a `settings` dir unless `Global` is necessary to differentiate from other settings.

Reply:
`Global` isn't really needed, it's just what the text in the design labels it.